substrate/frame/examples/frame-crate/Cargo.toml

```
@@ -0,0 +1,24 @@
+[package]
+name = "pallet-example-frame-crate"
+version = "4.0.0-dev"
```

```suggestion
version = "0.0.1"
```
I cant see these random numbers anymore :see_no_evil: